### PR TITLE
Template fixes

### PR DESCRIPTION
**Pull request by parithosh**

### CODE REVIEW

File: terraform/devnet-0/ansible_inventory.tmpl [hunk 1]
```diff
@@ -1,111 +1,57 @@
+localhost
+
 [all:vars]
-ethereum_network_name=${ethereum_network_name}
+ethereum_network_name=${sepolia_shadowfork_naming_override}
```
Comment (barnabasbusa) [inline terraform/devnet-0/ansible_inventory.tmpl:4]: This does not look ok.

File: terraform/devnet-0/main.tf [hunk 2]
```diff
@@ -440,10 +440,15 @@ resource "local_file" "ansible_inventory" {
...
+                index([for v in var.digitalocean_vm_groups : v.id], split(".", key)[0])
+              ].vms[split(".", key)[1]].ansible_vars
+            , null)
           }
         }
       )
     }
   )
-  filename = "../../ansible/inventories/devnet-0/inventory.ini"
+  filename = "../../ansible/inventories/devnet-2m-test/inventory.ini"
```
Comment (barnabasbusa) [inline terraform/devnet-0/main.tf:453]: We shouldn’t change the defaults

File: terraform/devnet-0/ansible_inventory.tmpl (hunk 1 above)
Comment (parithosh) [reply terraform/devnet-0/ansible_inventory.tmpl]: fixed, forgot to backport this fix

File: terraform/devnet-0/main.tf (hunk 2 above)
Comment (parithosh) [reply terraform/devnet-0/main.tf]: agreed, made a mistake backporting fixes. fixed

**Review by barnabasbusa — APPROVED** [review]
Lgtm

**Pull request merged**